resources/views/test-unit-notification.blade.php

```diff
@@ -0,0 +1,19 @@
+/* @@covers \{{ $covered }} */
+
+use {{ $classFqcn }};
+
```

Comment:
add a test for the via method

Reply:
Done in d52c406e896212317bac1cd776ad3a3bdd627312